introduction.md

```diff
@@ -4,3 +4,9 @@ I am ninja coder
 
 My favorite CodeSchool Path so far has been `JavaScript`
 
+Deadly Skills
+-------------
+* Git
+* JavaScript
+* ColdFusion
+
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?